### PR TITLE
Move to IMSC 1.2 ED

### PR DESCRIPTION
**Pull request by palemieux**

### CODE REVIEW

File: imsc1/spec/ttml-ww-profiles.html [hunk 1]
```diff
@@ -4704,16 +4707,16 @@ <h3>CFF-TT</h3>
     </section>
 
     <section class='appendix' id='imsc-interop'>
-      <h3>IMSC1 and IMSC1.0.1</h3>
+      <h3>Other versions of IMSC</h3>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: Perhaps: s/`Other`/`Previous` ? This section can't talk about interop with future versions.

File: imsc1/spec/ttml-ww-profiles.html [hunk 2]
```diff
@@ -4704,16 +4707,16 @@ <h3>CFF-TT</h3>
...
 
-      <p>The <a>Image Profile</a> is a strict superset of the [[ttml-imsc1]] and [[ttml-imsc1.0.1]] Image Profile.</p>
+      <p>The <a>Image Profile</a> is a strict superset of the [[ttml-imsc1]], [[ttml-imsc1.0.1]] and [[ttml-imsc1.1]] Image Profile.</p>
 
-      <p>A document that conforms to [[ttml-imsc1]] or [[ttml-imsc1.0.1]] therefore also conforms to this specification.</p>
+      <p>A document that conforms to [[ttml-imsc1]] or [[ttml-imsc1.0.1]] or [[ttml-imsc1.1]] therefore also conforms to this specification.</p>
 
-      <p>A number of features that were previously permitted in [[ttml-imsc1.0.1]] are designated as permitted-deprecated in the
-      <a>Text Profile</a> and <a>Image Profile</a>. [[ttml-imsc1.0.1]] documents that use such features are therefore susceptible
+      <p>A number of features that were previously permitted in other versions of IMSC are designated as permitted-deprecated in the
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: again, s/other/previous

File: imsc1/spec/ttml-ww-profiles.html [hunk 3]
```diff
@@ -191,28 +190,24 @@
 
 <body>
   <section id='abstract'>
-    <p>This specification defines two profiles of [[!ttml2]]: a text-only profile and an image-only profile. These profiles are
+    <p>This specification defines two profiles of [[ttml2]]: a text-only profile and an image-only profile. These profiles are
```
Comment (cconcolato) [inline imsc1/spec/ttml-ww-profiles.html]: Why is TTML2 not a normative reference anymore?

File: imsc1/spec/ttml-ww-profiles.html (hunk 1 above)
Comment (palemieux) [reply imsc1/spec/ttml-ww-profiles.html]: Addressed as suggested.

File: imsc1/spec/ttml-ww-profiles.html (hunk 3 above)
Comment (palemieux) [reply imsc1/spec/ttml-ww-profiles.html]: TTML2 remains a normative reference elsewhere, but is marked informative in the abstract, which is an informative section.

**Pull request merged**